test/service_manager_test.dart

```diff
@@ -109,98 +109,39 @@ void main() {
     });
 
     test('callService', () async {
-      dynamic _e;
-
       final registeredService =
           serviceManager.methodsForService['reloadSources'] ?? const [];
       expect(registeredService, isNotEmpty);
 
-      try {
-        await serviceManager.callService('reloadSources',
-            isolateId: serviceManager.isolateManager.selectedIsolate.id);
-      } catch (e) {
-        _e = e;
-      }
-
-      expect(_e, isNull);
+      await serviceManager.callService('reloadSources',
+          isolateId: serviceManager.isolateManager.selectedIsolate.id);
```

Comment:
nit: add a trailing comma to get the flutter style for these calls which I think looks a little nicer.

Reply:
done.